.github/workflows/build-dev.yaml

```diff
@@ -42,26 +42,9 @@ jobs:
     - name: Push to Docker Hub
       run: |
         set -e; ver=${{ env.BUILD_VERSION }}
-        docker tag defichain-x86_64-pc-linux-gnu:${ver} defi/defichain:${ver}
+        docker tag defichain-x86_64-pc-linux-gnu:${ver} defi/defichain:${ver}-linux-x64
```

Comment:
This needs to be the same tag as just the version, but correct platform. I think docker hub will auto resolve the platform if I'm right. 

Could you please verify pushing same tagged versions into Docker Hub?

Reply:
This is correct. Resolved in commit 8deeff23e